lib/pwwka/transmitter.rb

```diff
@@ -63,7 +63,7 @@ def self.send_message!(payload, routing_key,
       true
     rescue => e
 
-      logf "ERROR Transmitting Message on %{routing_key} -> %{payload}", routing_key: routing_key, payload: payload, at: :error
+      logf "ERROR %{error} Transmitting Message on %{routing_key} -> %{payload}", error: e, routing_key: routing_key, payload: payload, at: :error
```

Comment:
Can we add this to the end of the string?  I worry that there are log searches for "ERROR Transmitting Message" that this would break

Reply:
That is a good callout.

Reply:
Moved!